src/components/sbb-teaser-hero/sbb-teaser-hero.stories.js

```diff
@@ -1,71 +1,128 @@
+    <span slot="icon">{getMarkupForSvg('chevron-small-right-small')}</span>
+  </sbb-link>
+);
+
+const TemplateSbbPanelDefault = (args) => (
+  <sbb-panel slot="panel" {...args}>
+    <SlotSbbPanelTextTemplate {...args} />
+    <SlotPanelSbbLinkTemplate {...sbbPanelLinkArgs} />
+  </sbb-panel>
+);
+
+const sbbTeaserHeroImageArgs = {
+  imageSrc:
+    'https://cdn.img.sbb.ch/content/dam/internet/sharedimages/personen/frau-im-ferien.jpg?crop=focalpoint&fp-x=0.5053125&fp-y=0.6458333&fp-z=1&w=1536&h=960&auto=format,compress,cs=tinysrgb&q=30',
+};
+
+const SlotTeaserHeroImageTemplate = (args) => (
+  <img slot="image" src={args.imageSrc} alt="SBB CFF FFS Angestellte" />
+);
+
 // --- Component
 
-const Template = (args) => <sbb-teaser-hero {...args} />;
+const TemplateSbbTeaserHeroDefault = (args) => (
+  <sbb-teaser-hero {...args}>
+    <TemplateSbbPanelDefault {...args} />
+    <SlotTeaserHeroImageTemplate {...sbbTeaserHeroImageArgs} />
+  </sbb-teaser-hero>
+);
+
+const TemplateSbbPanelWithoutLink = (args) => (
+  <sbb-teaser-hero {...args}>
+    <sbb-panel slot="panel" {...args}>
+      <SlotSbbPanelTextTemplate {...args} />
+    </sbb-panel>
+    <SlotTeaserHeroImageTemplate {...sbbTeaserHeroImageArgs} />
+  </sbb-teaser-hero>
+);
 
-export const defaultTeaser = Template.bind({});
-export const openInNewWindow = Template.bind({});
+export const defaultTeaser = TemplateSbbTeaserHeroDefault.bind({});
+export const openInNewWindow = TemplateSbbTeaserHeroDefault.bind({});
+export const withoutLink = TemplateSbbPanelWithoutLink.bind({});
 
 defaultTeaser.argTypes = {
-  'image-loading': imageLoading,
   'open-in-new-window': openInNewWindowControl,
+  text: panelText,
 };
 
 defaultTeaser.args = {
-  'button-text': 'Button text',
-  'image-loading': imageLoading.options[0],
-  'image-src': images[0],
   link: 'https://www.sbb.ch',
-  'new-window-info-text': '',
+  accessibilityTitle: 'sbb-teaser-hero label',
+  text: 'Rücksichtsvoll mit SBB Green Class',
   'open-in-new-window': openInNewWindowControl.options[1],
-  text: 'Panel text',
 };
 
 defaultTeaser.documentation = {
   title: 'Default Teaser',
 };
 
 openInNewWindow.argTypes = {
-  'image-loading': imageLoading,
   'open-in-new-window': openInNewWindowControl,
+  text: panelText,
 };
 
 openInNewWindow.args = {
-  'button-text': 'Button text',
-  'image-loading': imageLoading.options[0],
-  'image-src': images[0],
+  text: 'Rücksichtsvoll mit SBB Green Class',
   link: 'https://www.sbb.ch',
   'new-window-info-text': 'Link öffnet in neuem Fenster.',
   'open-in-new-window': openInNewWindowControl.options[0],
-  text: 'Panel text',
+};
+
+withoutLink.argTypes = {
+  text: panelText,
+};
+
+withoutLink.args = {
+  text: 'Rücksichtsvoll mit SBB Green Class',
 };
 
 openInNewWindow.documentation = {
   title: 'Teaser Link open in new Window',
 };
 
 export default {
+  decorators: [
+    (Story) => (
+      <div>
```

Comment:
can you add some space so the image doesn't stick to the borders of the storybook box?